examples/with-universal-configuration-build-time/next.config.js

```diff
@@ -0,0 +1,10 @@
+require('dotenv').config()
+
+const prod = process.env.NODE_ENV === 'production'
+
+module.exports = {
+  env: {
+    TEST: process.env.TEST,
```

Comment:
I wonder if the result of `.config` should be used to automatically pass through values.

Reply:
https://github.com/motdotla/dotenv#config

Reply:
I did think about that, but since that would expose all the values in `.env` automatically to the clientside I felt iffy about it?

Reply:
Yeah I have the same concern. I think your solution right now is fine as long as we document it well 👍

Reply:
I'll update this file with a commented-out example of how to include all variables and a warning. I will also include a link to `with-dotenv` in the Readme and explain the difference.

Reply:
Sounds like a great solution 👍

Reply:
Fixed in 4cfb265

Example did become a bit verbose since some logic was needed to resolve values from `process.env` to handle overrides. Do you think it's clear enough?

Reply:
I just realised a potential alternative would be to put this in a `next.alternative.config.js`-file. What do you think makes the most sense?

Reply:
I think this looks great to be honest 👍